tagstudio/tag_studio.py

```diff
@@ -4,9 +4,14 @@
 
 """TagStudio launcher."""
 
+import os
+import sys
+
+sys.path.insert(
+    0, os.path.abspath(os.path.join(os.path.dirname(__file__)))
+)  # add this so that `poetry run tagstudio` works
```

Comment:
Didn't you add an entrypoint console script? I'm not sure if this is a solution to the problem or just a solution to a xy problem. Can you elaborate about this please?

Reply:
See my [following comment](https://github.com/TagStudioDev/TagStudio/pull/368#issuecomment-2323405866) or [ModuleNotFoundError for poetry script](https://github.com/python-poetry/poetry/issues/3928#issuecomment-1399313433) for more information.

Reply:
I did add an entry point to the script. But this project uses . relative imports so you need to fix that using this. Of course I could not have moved away from these relative imports. You can also have this problem when adding an extra py script in the root it is about where the working directory is. Something you can not change in the entry point definition either. 

I think adding the module to the import path is an elegant solution. This way it will just always work, not just for poetry.